app/src/main/java/org/wikipedia/savedpages/SavedPageSyncService.java

```diff
@@ -237,33 +246,47 @@ private long savePageFor(@NonNull ReadingListPage page) throws Exception {
                 .flatMap(response -> Observable.just(response.getDisplayTitle()))
                 .onErrorReturnItem(pageTitle.getDisplayText()); // prevent "redirected" or variant issue
 
+        // TODO: remove page lead and maybe load page summary since it needs description for page
         Observable<retrofit2.Response<PageLead>> leadCall = reqPageLead(CacheControl.FORCE_NETWORK, OfflineCacheInterceptor.SAVE_HEADER_SAVE, pageTitle);
```

Comment:
Right, the `PageLead` endpoint should no longer be used, and can be replaced with `Summary`.

Reply:
Done!